packages/element-lib/test/Truffle-Tests/Element Contract Test Explainer.md

```diff
@@ -0,0 +1,90 @@
+# Element Contract Test Explainer
```

Comment:
Not a fan of having spaces in the name of this file. I suggest renaming to element-contract-test-explainer.md

Reply:
will change

Reply:
done with new commit @gjgd